libraries/storage/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+set(HEADERS
+    include/storage/storage.hpp
+)
+
+set(SOURCES
+    src/storage.cpp
+)
+
+add_library(storage ${SOURCES} ${HEADERS})
+target_include_directories(storage PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include)
+target_link_libraries(storage PUBLIC CONAN_PKG::rocksdb)
+target_link_libraries(storage PRIVATE 
+    common
+    config
+    logger
+    pbft_block
+    proposal_period_levels_map
```

Comment:
does this really needs to be separate lib?

Reply:
That was a part of the dag. But it is needed for `storage` lib. The closest thing it could be merged with is `dag_block`. So I can do it if you want

Reply:
yeah I think I would go with moving it into the dag_block